openedx/core/djangolib/tests/test_js_utils.py

```diff
@@ -65,6 +68,31 @@ def test_dump_js_escaped_json_with_custom_encoder_escapes_unsafe_html(self):
         escaped_json = dump_js_escaped_json(malicious_dict, cls=self.SampleJSONEncoder)
         self.assertEquals(expected_custom_escaped_json, escaped_json)
 
+    def test_dump_js_escaped_json_with_lazy_text(self):
+        """
+        Verify that the function dump_js_escaped_json can process a django lazy text.
+        Taking translatable text of platform name as a sample
+        """
+        lazy_text = _('Your Platform Name Here')
```

Comment:
`_('Your Platform Name Here')` -> `ugettext_lazy('P𝓵𝓪tform 𝓝ame')`, just to add a little Unicode magic for extra testing.

Wondering where did I get those Unicode chars from? It's here: http://tholman.com/unicoder.js/